krux_boto/util.py

```diff
@@ -43,6 +43,25 @@ def get_instance_region():
         raise Error('get_instance_region failed to get the local instance region')
     return zone.rstrip(string.lowercase)
 
+def setup_hosts(hosts, accepted_domains=['krxd.net'], default='krxd.net'):
```

Comment:
please pull strings out of the code and into variables at the top. @hanpeter can probably point you at the style we use for this.